modules/gapi/src/backends/ie/giebackend/gieapi.hpp

```diff
@@ -0,0 +1,159 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+#ifndef OPENCV_GAPI_IEAPI_HPP
```

Comment:
`giewrapper.hpp`

Reply:
if you decide to go with a C++ file too, + `giewrapper.cpp`

Reply:
done